Resources/views/Admin/Security/Resetting/request.html.twig

```diff
@@ -0,0 +1,50 @@
+{% extends base_template %}
+
+{% block sonata_nav %}
+{% endblock sonata_nav %}
+
+{% block logo %}
+{% endblock logo %}
+
+{% block sonata_left_side %}
+{% endblock sonata_left_side %}
+
+{% block body_attributes %}class="sonata-bc login-page"{% endblock %}
+
+{% block sonata_wrapper %}
+
+<div class="login-box">
```

Comment:
do you think you are able to open a separate PR to fix this "anomaly" in the ```login.html.twig```?

this would be awesome :+1: